config/benchmark_examples/detection-coco2017-yolox_s/model.yaml

```diff
@@ -26,11 +26,10 @@ model:
       name: anchor_free_decoupled_head
       params:
         act_type: *act_type
-        # postprocessor - decode
-        score_thresh: 0.01
-        # postprocessor - nms
-        nms_thresh: 0.65
-        class_agnostic: False
+  postprocessor: 
```

Comment:
Please keep comments in configuration files

Reply:
Alright! I'll re-upload PR soon!